api/src/main/java/org/apache/gravitino/credential/GCSTokenCredential.java

```diff
@@ -70,4 +83,11 @@ public Map<String, String> credentialInfo() {
   public String token() {
     return token;
   }
+
+  private void validate(String token, long expireTimeInMs) {
```

Comment:
How about embed this method into `initialize`?
The GCSTokenCredential is immutable IIUC, so I suppose that this validation
would be invoked only from `initialize`.
We can move this out when this assumption doesn't hold in the future.

Reply:
This is out of scope of this PR as it's outside the CLI code or any changes made by this PR.  Best to raise a new issue for it.